Resources/doc/reference/dashboard.rst

```diff
@@ -50,6 +50,8 @@ Configuring the ``Admin`` list
 
 As you probably noticed by now, the ``Admin`` list groups ``Admin`` mappings together.
 There are several ways in which you can configure these groups.
+The order of all admins is when you defined them. To get a natural sorting add the option
+admins_sort: true.
```

Comment:
which config file? I already added it to Resources/doc/reference/configuration.rst